src/domain/auth/authorization.repository.interface.ts

```diff
@@ -8,4 +11,21 @@ export interface IAuthorizationRepository {
     signerAddress: `0x${string}`;
     authPayload: AuthPayload | undefined;
   }): void;
+
+  assertSafeOwner(args: {
+    safeAddress: `0x${string}`;
+    authPayload: AuthPayload | undefined;
+  }): Promise<void>;
 }
+
+@Module({
+  imports: [SafeRepositoryModule],
+  providers: [
+    {
+      provide: IAuthorizationRepository,
+      useClass: AuthorizationRepository,
+    },
+  ],
+  exports: [IAuthorizationRepository],
+})
+export class AuthorizationRepositoryModule {}
```

Comment:
How does this one compare to `AuthRepositoryModule`?

Reply:
In hindsight, they do not differ much. I've moved these methods into `AuthRepository`, negating the need for this module in 48b9b811b02a9b83e2c15e45bb3839e4d1d0caad.